websrc/default_hands/src/default_hands_main.tsx

```diff
@@ -960,6 +1017,16 @@ class DefaultHands extends React.Component< {}, {} >
 			}
 		};
 
+		if (this.state.menuGestureCollideMain && this.state.menuGestureCollideSecondary && Date.now() >= this.menuGestureCooldownCounter + 500)
+		{
+			this.menuGestureCooldownCounter = Date.now();
+			this.toggleGadgetMenu();
+			this.displayIntro = false;
+		}
+
+		this.controllerType = inputProcessor.currentInteractionProfile ?? "/interaction_profiles/valve/index_controller"; // we default to index just incase something goes wrong
```

Comment:
I was going to suggest renaming this member, but why cache the value here at all? Can't you just read it out of inputProcessor and then store it in a local?